components/Footer/Footer.jsx

```diff
@@ -1,5 +1,87 @@
-const Footer = (props) => {
-    const { children } = props
-    return <footer>{children}</footer>
+import Image from 'next/image'
+import britoFlag from '../../public/images/brito-logo-1.svg'
+import britoWordMark from '../../public/images/brito-logo-2.svg'
+import styles from '@/styles/Footer.module.css'
+import labels from '@/constants/labels.en'
+import { Icon } from '@/components'
+
+const Footer = () => {
```

Comment:
Please pass the content using props: 
Example:
- description
- menus 
- copyright
- secondary menu